packages/aura/src/components/menu-bar.css

```diff
@@ -15,3 +15,7 @@ vaadin-menu-bar-button[slot='overflow'] {
   font-family: system-ui, sans-serif;
   font-weight: 600;
 }
+
+vaadin-menu-bar-submenu {
+  --vaadin-overlay-backdrop-background: transparent;
```

Comment:
IMO we should have this in base styles, similarly as we did for `vaadin-select`. Also, should we have it consistently for both context-menu and menu-bar?